ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -547,62 +449,43 @@ void OwInterface::initialize()
   static bool initialized = false;
 
   if (not initialized) {
+    LanderInterface::initialize();
 
```

Comment:
Why is this needed instead of just calling `LanderInterface::initialize()`? Is there a chance `OwInterface::initialize` can be called more than once?

Reply:
You're right, the guard is not needed since initialize() should never be called more than once.  I'll remove it.